azure-pipelines.yml

```diff
@@ -20,8 +20,8 @@ variables:
   PathToCommunityToolkitMediaElementAnalyzersCodeFixCsproj: 'src/CommunityToolkit.Maui.MediaElement.Analyzers.CodeFixes/CommunityToolkit.Maui.MediaElement.Analyzers.CodeFixes.csproj'
   PathToCommunityToolkitAnalyzersUnitTestCsproj: 'src/CommunityToolkit.Maui.Analyzers.UnitTests/CommunityToolkit.Maui.Analyzers.UnitTests.csproj'
   DotNetMauiRollbackFile: 'https://maui.blob.core.windows.net/metadata/rollbacks/7.0.49.json'
-  Latest_Xcode_Version: '14.1.0'
-  CommunityToolkit_Xcode_Version: '14.0.1'
+  Latest_Xcode_Version: '14.2.0'
+  CommunityToolkit_Xcode_Version: '14.2.0'
```

Comment:
We need to keep the Community Toolkit 
Xcode version pinned to the same version of Xcode that was supported when .NET v7.0.100 shipped. 

I don't believe net-ios supported Xcode 14.2.0 when .NET v7.0.100 shipped

Reply:
We can bump this Xcode version when we bump our .NET version

Reply:
We might need to add some kind of comment on why that is I guess then because I have no clue why there are two separate versions in here. It seems a bit weird 😅

Reply:
One should be the latest (the most recent) supported release of Xcode. 

The other is the version of Xcode that we use to build the toolkit.